.gitignore

```diff
@@ -1,12 +1,13 @@
-.idea
 .DS_Store
-node_modules
-archives/
-yarn-error.log
+.eslintcache
+.idea
 .pnpm-debug.log
-server*.log
-*.png
-*.jpg
-*.jpeg
```

Comment:
any reason for some of this deletion ?

Reply:
i think the deleted lines are just moved.